woocommerce/woocommerce
# On-/offboarding copy updates

### Submission Review Guidelines:

-   I have followed the [WooCommerce Contributing Guidelines](https://github.com/woocommerce/woocommerce/blob/trunk/.github/CONTRIBUTING.md) and the [WordPress Coding Standards](https://make.wordpress.org/core/handbook/best-practices/coding-standards/).
-   I have checked to ensure there aren't other open [Pull Requests](https://github.com/woocommerce/woocommerce/pulls) for the same update/change.
-   I have reviewed my code for [security best practices](https://developer.wordpress.org/apis/security/).
-   Following the above guidelines will result in quick merges and clear and detailed feedback when appropriate.



### Changes proposed in this Pull Request:





Closes #38960



### How to test the changes in this Pull Request:



Using the [WooCommerce Testing Instructions Guide](https://github.com/woocommerce/woocommerce/wiki/Writing-high-quality-testing-instructions), include your detailed testing instructions:

New users flow. Note that for this flow the store should not has any published product.
1. Make sure the `New product editor` feature is active under `/wp-admin/admin.php?page=wc-settings&tab=advanced&section=features`
2. Make sure `woocommerce_block_product_tour_shown` is set to `no` and `woocommerce_product_editor_show_feedback_bar` is set to `yes` under `/wp-admin/tools.php?page=woocommerce-admin-test-helper` so `WCA Test Helper` should be installed
3. Visit `/wp-admin/admin.php?page=wc-admin&path=/add-product` you should see the `welcome modal` with title `Meet the product editing form` and description `Discover the product form's unique features with a quick overview of what's included.`
4. After clicking `View highlights` button you should see a tour modal with title `Fresh and modern interface`, description `Everything you need to create and sell your products, all in one place. From photos and descriptions to pricing and inventory, all of your product settings can be found here.  `  and the illustration shared in second AC from #38960
5. Clicking the `Next` button a second modal should be shown with title `Content-rich product descriptions` and description `Show off what's great about your products and engage your customers with content-rich product descriptions. Add images, videos, and any other content they might need to make a purchase.`
6. Clicking the `Next` button a third modal should be shown with title `Lightning fast performance` and description `Get your products listed and available for purchase in no time! Our modern technology ensures a reliable and streamlined experience.`
7. Clicking the `Next` button a four modal should be shown with title `More features are on the way` and description `We're actively working on adding more features to the product form, including the ability to add digital products, variations, and more. Watch this space!`
8. After closing the last modal a bottom bar should slides out from the bottom of the screen and with some background blur. Check the design from #38960
9. Clicking `Share feedback` link from the bottom bar should show a feedback modal with tile `What do you think of the new product form?`
10. When clicking `turn it off` link from the bottom bar should show a new modal with the following checkboxes 
<img width="500" alt="image" src="https://github.com/woocommerce/woocommerce/assets/13334210/84bba30b-e8c3-44bd-b188-09f150e5fb8c"> and the primary button should be `Send`

Old users flow. Note that for this flow the store should has at least one published product.
1. Make sure the `New product editor` feature is active under `/wp-admin/admin.php?page=wc-settings&tab=advanced&section=features`
2. Make sure `woocommerce_block_product_tour_shown` is set to `no` and `woocommerce_product_editor_show_feedback_bar` is set to `yes` under `/wp-admin/tools.php?page=woocommerce-admin-test-helper` so `WCA Test Helper` should be installed
3. Visit `/wp-admin/admin.php?page=wc-admin&path=/add-product` you should see the `welcome modal` with title `Welcome to the new product form!` and description `Discover its new features and improvements with a quick overview of what's included.`



### Changelog entry




<details>

#### Significance




#### Type




#### Message 

#### Comment 

</details>
